src/server/src/routes/job.ts

```diff
@@ -0,0 +1,82 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response, Router } from 'express';
+import { IJob } from '@entities/job';
+import { errors } from '@shared/errors';
+import {
+    createJob,
+    getJobs,
+} from '@modules/job';
+import logger from '@shared/Logger';
+
+const router = Router();
+
+const { BAD_REQUEST, CREATED, OK, INTERNAL_SERVER_ERROR } = StatusCodes;
+
+interface jobRequest extends Request {
+    body: {
+        job: IJob;
+    };
+}
+
+/******************************************************************************
+ *            POST Request example - Create - "POST /api/job/create"
+ ******************************************************************************/
+
+router.post('/create', async (req: jobRequest, res: Response) => {
+    const { job } = req.body;
+    const {
+        targetYears,
+        hoursPerWeek,
+        description,
+        startDate,
+        endDate,
+        type,
+        title,
+        status,
+        minSalary,
+        maxSalary } = job;
+    if (!job) {
+        return res.status(BAD_REQUEST).json({
+            error: errors.paramMissingError,
+        });
+    }
+    try {
+        await createJob(targetYears,
+                        hoursPerWeek,
+                        description,
+                        startDate,
+                        endDate,
+                        type,
+                        title,
+                        status,
+                        minSalary,
+                        maxSalary);
+        return res.status(CREATED).end();
+    } catch (error) {
+        logger.err(error);
+        return res
+            .status(INTERNAL_SERVER_ERROR)
+            .json(errors.internalServerError)
+            .end();
+    }
+});
+
+router.get('/read', async (req: Request, res: Response) => {
+    try {
+        const jobs = await getJobs();
+        console.log(jobs);
+        return res.status(OK).json(jobs).end();
```

Comment:
should be **return res.status(OK).json({ jobs }).end();** since we want to return a variable called jobs that will store an array of jobs. Without this, you are just sending the array of jobs without an identifier.